docs: add some tips on collecting per-queue metrics for RabbitMQ (#12227)
netdata/netdata: collectors/python.d.plugin/rabbitmq/README.md

---

> ### Per-Queue Chart configuration
> RabbitMQ users with the "monitoring" tag cannot see all queue data, you'll need a user with read permissions here. To create a dedicated user for netdata:

Reviewer: ```suggestion
### Per-Queue Chart configuration

RabbitMQ users with the "monitoring" tag cannot see all queue data, you'll need a user with read permissions here. To create a dedicated user for netdata:
```

Reviewer: an empty line after the header, please

```bash
rabbitmqctl add_user netdata ChangeThisSuperSecretPassword
rabbitmqctl set_permissions netdata "^$" "^$" ".*"
```

> Once the user is set up, add `collect_queues_metrics: yes` to your config.

Reviewer: Let's add an actual example with user/pass/collect_queues_metrics

Author: good point, added :)

---

### Per-Queue Chart configuration

> RabbitMQ users with the "monitoring" tag cannot see all queue data, you'll need a user with read permissions here. To create a dedicated user for netdata:
> 

Reviewer 2: ```suggestion
RabbitMQ users with the "monitoring" tag cannot see all queue data. You'll need a user with read permissions. 
To create a dedicated user for netdata:

```

```bash
rabbitmqctl add_user netdata ChangeThisSuperSecretPassword
rabbitmqctl set_permissions netdata "^$" "^$" ".*"

> ```

Reviewer 2: Could you give a short explanation what these two lines will do?

> Once the user is set up, add `collect_queues_metrics: yes` to your config:

Reviewer 2: To the RabbitMQ config or the Netdata config?

> ```bash
> rabbitmqctl add_user netdata ChangeThisSuperSecretPassword
> rabbitmqctl set_permissions netdata "^$" "^$" ".*"
> Once the user is set up, add `collect_queues_metrics: yes` to your `rabbitmq.conf`:

Reviewer: ```suggestion
```bash
rabbitmqctl add_user netdata ChangeThisSuperSecretPassword
rabbitmqctl set_permissions netdata "^$" "^$" ".*"
```

See [set_permissions](https://www.rabbitmq.com/rabbitmqctl.8.html#set_permissions) for details.

Once the user is set up, add `collect_queues_metrics: yes` to your `rabbitmq.conf`:
```

---

### Per-Queue Chart configuration

RabbitMQ users with the "monitoring" tag cannot see all queue data. You'll need a user with read permissions. 
To create a dedicated user for netdata:

```bash
rabbitmqctl add_user netdata ChangeThisSuperSecretPassword
rabbitmqctl set_permissions netdata "^$" "^$" ".*"

> ```

Reviewer: ```suggestion
```

See [set_permissions](https://www.rabbitmq.com/rabbitmqctl.8.html#set_permissions) for details.

```

> Once the user is set up, add `collect_queues_metrics: yes` to your config:

Reviewer: ```suggestion
Once the user is set up, add `collect_queues_metrics: yes` to your `rabbitmq.conf`:
```
